js/parts/Series.js

```diff
@@ -4971,7 +4972,7 @@ null,
         if (!hasRendered) {
             // Additional time if defer is defined before afterAnimate
             // will be triggered
-            if (animDuration && defined(animObject(options.animation).defer)) {
+            if (animDuration && (animObject(options.animation).defer)) {
```

Comment:
`(animObject(options.animation).defer)` -> `animObject(options.animation).defer`

Reply:
I will fix it, but it is not the final code, because how this logic will look like depends on the above conversation about implementing the defer inside the **animObject**.